### DISCUSSION TITLE
Creating new clickable button to perform certain actions on Github Pull Request

### DISCUSSION THREAD

**Discussion by Sujathan18** [community/community#18011] [Pull Requests]
Hello Git Community,

I am having below requirement, implement mechanism to achieve the same,

When a Pull request is raised in GitHub, a Jenkins job will be triggered => This is already implemented.

New Requirement to be implemented:
1. When the above Jenkins build fails, a new clickable **button** needs to created and enabled on the Pull request
2. Upon clicking the button some actions needs to be performed.

Need any sample code or ways to implement it. Need the complete workflow guide on how to add this button, where to add the code and how to make the button appear on the PR all these details as I am new to GitHub.

**Comment by 13steinj** [comment]
The simplest way that I can think of would be to create a comment with links that have images on them, treat the comment as state, and upon click update the state and redirect back to the referred page.

**Comment by Nepo26** [comment]
Is there any update on how to perform those actions?
